diff --git a/contrib/svt-av1/module.defs b/contrib/svt-av1/module.defs
index 7c9116ecf..7796b2719 100644
--- a/contrib/svt-av1/module.defs
+++ b/contrib/svt-av1/module.defs
@@ -1,9 +1,8 @@
 $(eval $(call import.MODULE.defs,SVT-AV1,svt-av1))
 $(eval $(call import.CONTRIB.defs,SVT-AV1))
 
-SVT-AV1.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs2/SVT-AV1-v4.2.0.tar.gz
-SVT-AV1.FETCH.url    += https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v4.2.0/SVT-AV1-v4.2.0.tar.gz
-SVT-AV1.FETCH.sha256  = c7b13c4a84bd3751aa35fcc72be13e6875467e7c2216879251a486e5b1e4e740
+SVT-AV1.FETCH.url     = https://github.com/Uranite/HandBrake-SVT-AV1-Tritium/releases/download/contribs/svt-av1-tritium.tar.gz
+SVT-AV1.FETCH.sha256  = 05a9c2d9aa127bca8303cbf8ecb31d2b99a51f706e92522ca3d17b26bbc4db7a
 
 SVT-AV1.GCC.args.c_std =
 
